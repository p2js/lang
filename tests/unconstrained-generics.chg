function foo<T>(): i32 {
    return 42;
}

function main(): i32 {
    return foo();
}
