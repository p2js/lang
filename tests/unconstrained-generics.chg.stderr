

    return foo();
           ^^^^^  type error: unconstrained type variable created here

    switch (ty.type) {
               ^

TypeError: Cannot read properties of null (reading 'type')








    
    


